lib/typescript/httpclient/payload/PaginatedPayload.ts

```diff
@@ -1,4 +1,5 @@
+import {ResponseMetadataPayload} from '../payload/ResponseMetadataPayload';
 export interface PaginatedPayload<T> {
   data: T[];
-  responseMetadata: {previousCursor: string; nextCursor: string; total: number};
+  responseMetadata: ResponseMetadataPayload;
```

Comment:
this type was also not introduced on purpose... it's not really needed right?

Reply:
why is not needed?

Reply:
because it's not really its own thing... it's always embedded in the paginatedpayload that is already a type...